migrate/migrations/20250302/script.sh

```
@@ -0,0 +1,6 @@
+title "Update codeowners file"
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)